service/src/main.rs

```diff
@@ -16,77 +16,7 @@
 */
 
 #![cfg_attr(test, feature(assert_matches))]
-
```

Comment:
Extracted the whole file into a `main_impl` I thought that the overview is better then.